Manager/src/main/java/io/raspberrywallet/manager/bitcoin/Bitcoin.java

```diff
@@ -193,7 +199,8 @@ private void synchronizeWalletNonBlocking(final Wallet wallet, @Nullable String
                 try {
                     saveEncryptedWallet(password);
                     Logger.info("Wallet balance" + wallet.getBalance().toFriendlyString());
-                } catch (IOException | WalletNotInitialized e) {
+                } catch (IOException | WalletNotInitialized | IncorrectPasswordException e) {
```

Comment:
IncorrectPasswordException should be separated from others and doesn't print stack trace